### DISCUSSION TITLE
JPEG Quality Applied Only to New Images

### DISCUSSION THREAD

**Discussion by Pilotwings-64** [TriliumNext/Trilium#2698] [Q&A] [answered]
I did some tests, changing the image quality setting and pasting images into notes. I notice that the current JPEG quality setting was applied only to images pasted into Trilium after that setting was applied. Images pasted in before the setting was applied were not affected by the new setting. Even after closing and reopening Trilium, this remained true.

Is this true across the board in Trilium, meaning there will be no obscure circumstance in which the new JPEG quality setting will be applied to old images already in the database?

Can I rely on the same thing to be true, even with future versions of Trilium?

Thanks. I love this software.

**Comment by zadam** [comment, marked as answer]
Yes, that's the behavior. Trilium does not recompress images after they are uploaded. 

> Can I rely on the same thing to be true, even with future versions of Trilium?

Yes, this is not going to change.

**Reply by Pilotwings-64** [reply]
Thank you so much!

Is there a way to change that setting with a button, or does it have to be done from the settings menu?
